docs/dapps/developer-toolchains/using-foundry-with-the-avalanche-c-chain.md

```diff
@@ -133,14 +132,33 @@ generated by running:
 forge init --no-git --no-commit --force
 ```
 
+The `foundry.toml` by default points to the folders it added.
+We will want to change this to make sure the
+`src` points to the `contracts` directory.
+Change your `foundry.toml` to look like the following:
+
+```zsh
+[profile.default]
+src = 'contracts'
+out = 'out'
+libs = ["node_modules", "lib"]
+remappings = [
+    '@ensdomains/=node_modules/@ensdomains/',
+    '@openzeppelin/=node_modules/@openzeppelin/',
+    'hardhat/=node_modules/hardhat/',
```

Comment:
This isn't exactly the "Foundry" way of doing things and might be a little confusing to be honest